Easy_Problems/Coderbyte16.js

```diff
@@ -0,0 +1,16 @@
+function ArithGeo(arr) {
```

Comment:
Doesn't make sense

* first you are always returning in the for loop no matter what.  So the loop while always only run 1 time
* second it will always return "Arithmetic" no matter what because the first loop with i = 0 is equal the diff
* 3rd you aren't calling the function correctly